docs/sources/setup-grafana/set-up-https.md

```diff
@@ -237,7 +237,7 @@ To configure Grafana HTTPS and restart Grafana, complete the following steps.
 
    > **Note**: The standard port for SSL traffic is 443, which you can use instead of Grafana's default port 3000. This change might require additional operating system privileges or configuration to bind to lower-numbered privileged ports.
 
-1. [Restart the Grafana server]({{< relref "./start-restart-grafana/#linux" >}}) using `systemd`, `init.d`, or the binary as appropriate for your environment.
+1. [Restart the Grafana server]({{< relref "./restart-grafana/#linux" >}}) using `systemd`, `init.d`, or the binary as appropriate for your environment.
```

Comment:
```suggestion
1. [Restart the Grafana server]({{< relref "./restart-grafana#linux" >}}) using `systemd`, `init.d`, or the binary as appropriate for your environment.
```

It's convenient to avoid the trailing slash because that means the link won't break if the destination file is changed between being a page (`page.md`), leaf bundle (`page/index.md`), or a branch bundle (`page/_index.md`).

- https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash